src/draive/gemini/lmm_generation.py

```
@@ -127,142 +141,454 @@ async def lmm_completion(  # noqa: C901, PLR0912, PLR0915
+                                    "Invalid Gemini completion - exceeded maximum length!",
+                                    completion_chunk,
+                                )
+
+                            case reason:  # other cases are errors
+                                raise GeminiException(
+                                    f"Gemini completion generation failed! Reason: {reason}",
+                                    completion_chunk,
+                                )
+            except Exception:
+                # Re-raise any exceptions that occur during streaming
+                raise
```

Pylint (3.3.7)

[warning] The except handler raises immediately

(W0706)